src/test/kotlin/org/rust/ide/inspections/RsSortImplTraitMembersInspectionTest.kt

```diff
@@ -234,4 +245,32 @@ class RsSortImplTraitMembersInspectionTest : RsInspectionsTestBase(RsSortImplTra
             }
         }
     """)
+
+    fun `test does the right thing when the impl is missing some of the members`() = checkFixByText("Apply same member order", """
+        trait Foo {
+            type T;
+            const C: i32;
+            fn foo();
+        }
+
+        struct S;
+
+        <error descr="Not all trait items implemented, missing: `T` [E0046]">impl Foo for S/*caret*/</error> {
```

Comment:
Let's check that we show description of `RsSortImplTraitMembersInspection` too. You need to add `<weak_warning>` tag (see other tests for this inspection) and enable weak warning highlighting (pass the corresponding param to `checkFixByText ` function)

Reply:
Oh, I've just noticed that we don't enable weak warning checking in other tests so really we don't check inspection message at all.
Could you please fix it too?

Reply:
Sure! Done. 🙂